### PR TITLE
Fixing owner page after adding a new position

### PR DESCRIPTION
**Pull request by marcinciarka**
# [The my positions page is broken at the moment with this new position](https://app.shortcut.com/oazo-apps/story/7983/the-my-positions-page-is-broken-at-the-moment-with-this-new-position)

  
## Changes 👷‍♀️
- Fixing owner page when displaying Aave V3 position
  
## How to test 🧪
- `/owner/0x165d0df29d9262f8ee9e9874c906eab706a05c2c` - this has Aave V3 position

### CODE REVIEW

File: features/vaultsOverview/pipes/positions.ts [hunk 1]
```diff
@@ -125,18 +131,14 @@ type ProxyAddressesProvider = {
...
-  getAaveAssetsPrices$: (args: AaveV2AssetsPricesParameters) => Observable<BigNumber[]>
+  aaveV2: ProtocolsServices['AaveV2']
+  aaveV3: ProtocolsServices['AaveV3']
   tickerPrices$: (tokens: string[]) => Observable<Tickers>
-  wrappedGetAaveReserveData$: (token: string) => Observable<PreparedAaveReserveData>
-  aaveAvailableLiquidityInUSDC$: (reserveDataParameters: { token: string }) => Observable<BigNumber>
   automationTriggersData$: (id: BigNumber) => Observable<TriggersData>
 }
 
+type OnChainAavePositionTypes = 'AAVE_V2' | 'AAVE_V3'
```
Comment (jakubswierczek) [inline features/vaultsOverview/pipes/positions.ts:140]: Use the enum

**Review by jakubswierczek — CHANGES_REQUESTED** [review]
Great work,
The change with `ProtcolServices` is not very important, but please change the string type to an enum to have only one place with it.

File: features/vaultsOverview/pipes/positions.ts [hunk 2]
```diff
@@ -125,18 +131,14 @@ type ProxyAddressesProvider = {
...
 
 type BuildPositionArgs = {
-  aaveProtocolData$: (
-    collateralToken: string,
-    debtToken: string,
-    address: string,
-  ) => Observable<AaveProtocolData>
-  getAaveAssetsPrices$: (args: AaveV2AssetsPricesParameters) => Observable<BigNumber[]>
+  aaveV2: ProtocolsServices['AaveV2']
+  aaveV3: ProtocolsServices['AaveV3']
```
[the remark this answers is not in the document]
Comment (marcinciarka) [reply features/vaultsOverview/pipes/positions.ts:135]: Fixed.

File: features/vaultsOverview/pipes/positions.ts (hunk 1 above)
Comment (marcinciarka) [reply features/vaultsOverview/pipes/positions.ts:140]: Fixed.

**Review by jakubswierczek — APPROVED** [review]

**Pull request merged**